### DISCUSSION TITLE
date.formatDate return date wrong (one day of differ)

### DISCUSSION THREAD

**Discussion by ewertonrfd** [quasarframework/quasar#15416] [General - Components / Directives / etc]
Hello everyone,

**My Date**
<img width="167" alt="image" src="https://user-images.githubusercontent.com/13201575/217981424-363e2c24-2e4c-4156-a6f2-c12ff3dbfd19.png">

**Date differ using date.formatDate(val, "DD/MM/YYYY")**
<img width="217" alt="image" src="https://user-images.githubusercontent.com/13201575/217981519-13666822-2f33-405a-a3cd-924d0d3813e9.png">

**Without date.formatDate**
<img width="290" alt="image" src="https://user-images.githubusercontent.com/13201575/217981586-2d98a5d1-a135-442d-b433-356459ea896c.png">

For it to work I have to do it like this:
```
`${date.formatDate(date.addToDate(val, { hours: 3 }), "DD/MM/YYYY")}`,
```

Seeing here I realized that it's because of the timezone

**But would there be a way to disable it?**

Regards

**Comment by metalsadman** [comment]
First param must be a Date object.

**Reply by ewertonrfd** [reply]
I tried in some ways to pass other formats and I couldn't
From what I had seen in the doc, it would accept string int (timestamp) date (obj as you commented)

**Comment by ewertonrfd** [comment]
For now I used a function in my helpers just to do this formatting
Very simple the function

File

```ts
/**
  * Format date yyyy-mm-dd to dd/mm/yyyy
  * @param {string} date - Date to format
  * @return {string} Date formatted
  */
function formatDate(date) {
     return date.split("-").reverse().join("/");
}

export { formatDate };
```

Using

`import { formatDate } from "../helpers/formatDate";`

![image](https://user-images.githubusercontent.com/13201575/219035835-846b23de-227c-48a3-b848-82bc5362a26e.png)
